content/api/commands/clearcookies.md

```diff
@@ -40,8 +40,7 @@ Pass in an options object to change the default behavior of `cy.clearCookies()`.
 
 ### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
 
-<List><li>`cy.clearCookies()` yields `null`.</li><li>`cy.clearCookies()` cannot
-be chained further.</li></List>
+- `cy.clearCookies()` yields `null`.
```

Comment:
Is clearCookies a query or a command?

Reply:
Command. Also not sure it needs to be called out in the 'Yields' header though - it yields null, that's kind of the end of it. :woman_shrugging:

Reply:
Showed an example in a comment here - https://github.com/cypress-io/cypress-documentation/pull/4835#discussion_r1017145845

The problem is that "parent" and "child" commands do *not* work the way you expect them to from the name.

- Parent commands are those that *don't care about their subject* - being a parent does not limit where in a chain a command can be!
- Dual commands accept both null and a valid subject. They behave differently based on what subject they're given.
- Child commands require a subject. But this is a rule about the subject they're passed, not about their position. `cy.end().click()` is invalid because the subject is null, not because `.end()` can't be chained off of.

This is all existing behavior from Cy1-11, covered by our own tests - the docs have just never explained it clearly.

Reply:
```
  cy.visit('/fixtures/dom.html')
    .clearCookies('foo')
    .clearCookies('foo')
    .clearCookies('foo')
    .get('form')
    .get('form')
    .get('form')
```

Seven parent commands chained together. Our own tests do this all the time, as do our example repos. Forcing parents to actually be parents or disallowing chaining after a command that returns `null` would be a massively breaking change.

Reply:
The reason we use the term "yields" and not "returns" is because it's stating which subject is passed to the next command, not what is returned from the command. All commands return a chainer object that allows chaining further commands. 

While we do indeed use a lot of unnecessary chaining in our tests, there's at least one legitimate use for chaining after a command that yields `null`:

```js
cy.clearCookies().then(() => {
  // do something after clearing the cookies has completed
})
```